src/firebase/firebaseAuth.js

```diff
@@ -82,7 +82,7 @@ export function sendPasswordResetEmail(emailAddress, handleError) {
     });
 }
 
-export function logout(handleError) {
+export function logout(handleError, cart) {
```

Comment:
How come you've added this parameter? I don't see this getting used anywhere

Reply:
Ah, I think it's because you assume that we will need access to the cart on firebaseAuth.js as it will eventually need to store it into the DB. You won't need to do this, because Dom and Nick will make a new file called firebaseDB.js which is a whole separate function dedicated to storing/retrieving data. It's important that we isolate these functions that hold different responsibilities, rather than coupling them very tightly.

Reply:
With that said, can you please remove the cart parameter?

Reply:
One of my tasks for Monday was to make the cart accessible to the logout() function, that is what the parameter does. In navbar.jsx, I call the logout() function and pass in handleError() and this.props.cart.